### ISSUE TITLE
Inconsistent naming of the specs-go module

### ISSUE THREAD

**Issue by bart0sh** [cncf-tags/container-device-interface#227]
```
$ git grep 'container-device-interface/specs-go"'
pkg/cdi/cache.go:       cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/cache_test.go:  cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/container-edits.go:     "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/container-edits_test.go:        cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/device.go:      cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/device_test.go: cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/doc.go://       "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/doc.go://       "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/registry.go:    cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/spec.go:        cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/spec_test.go:   cdi "tags.cncf.io/container-device-interface/specs-go"
pkg/cdi/validate/schema.go:     raw "tags.cncf.io/container-device-interface/specs-go"
```

I'd propose to replace all of that with unnamed import `"tags.cncf.io/container-device-interface/specs-go"`

**Comment by klihub** [comment]
Isn't that consistent naming ? I mean everywhere within pkg/cdi, we call specs-go cdi, so the spec reference reads cdi.Spec, not specs.Spec. Similarly we call opencontainers/runtime-spec/spec-go oci, so we have then oci.Spec in the code.

Anyway you need to import one of them with a renamed alias, since you can't call both of them in the same module specs. So then probably you would have specs.Spec instead of cdi.Spec and maybe oci.Spec or ocispec.Spec for the oci Spec. Personally I consider both of these less readable than the current implementation choice.

**Comment by klihub** [comment]
Ah ok. In pkg/container-edits.go we're inconsistent. If the current and above described reasoning is acceptable, then at least that should be changed to be consistent with the rest within pkg/cdi. And then I'd change raw to cdi in pkg/cdi/validate/schema.go as well.

**Comment by klihub** [comment]
> Ah ok. In `pkg/container-edits.go` we're inconsistent... And then I'd change raw to cdi in pkg/cdi/validate/schema.go as well.

Filed #228 to fix those.

**Comment by bart0sh** [comment]
This seems to be closed by #228
/close
